pkg/tenant/projects.go

```diff
@@ -156,6 +156,15 @@ func (s *Server) TenantProjectCreate(c *gin.Context) {
 		Name:     project.Name,
 	}
 
+	if project.Description != "" {
+		if len(project.Description) > 2000 {
```

Comment:
We'll probably want to make this a constant so we can share it between api and db validation.